paramak/neutronics_utils.py

```diff
@@ -0,0 +1,121 @@
+
+import numpy as np
+
+
+def define_moab_core_and_tags():
+    """Creates a MOAB Core instance which can be built up by adding sets of
+    triangles to the instance
+
+    Returns:
+        (pymoab Core): A pymoab.core.Core() instance
+        (pymoab tag_handle): A pymoab.core.tag_get_handle() instance
+    """
+
+    try:
+        from pymoab import core, types
+    except ImportError as err:
+        raise err('PyMoab not found, export_h5m method not available')
```

Comment:
Do we need this here ? These lines will never be hit by the CI. Perhaps a simple import (without try except) is enough ?

Reply:
Just I would like the paramak to work for people who don't have pymoab installed so I thought it needed a bit of protection. I moved this section of code from the previous export_h5m statement which used this ImportError protection.